backend/framework/bin/exposure_monitoring.py

```diff
@@ -95,7 +95,7 @@ def run(self):
             self.process.start()
             self.running.set()
 
-        delete_exposures()
```

Comment:
Add this back so we cleanup after processing

Reply:
Sure! thanks